### PR TITLE
[REF] hw_{drivers, posbox_homepage}: status screen owl rewrite

### PR DESCRIPTION
**Pull request by MaxTheWhale**
Before this commit, `DisplayDriver_L` contained a
`/point_of_sale/display` endpoint. Historically,
this endpoint had been used for the PoS customer
display, but currently only acts as a status page
that is automatically displayed on the IoT box.
It is also the only page on the IoT box still
using Jinja for templating.

After this commit:
- The PoS display is renamed to Status display
- The page is rewritten using OWL
- The page is moved to `hw_posbox_homepage`

From a user perspective, the page remains
identical to before.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hw_drivers/connection_manager.py [hunk 1]
```diff
@@ -60,14 +58,6 @@ def _connect_to_server(self, url, token, db_uuid, enterprise_code):
         # Restart to checkout the git branch, get a certificate, load the IoT handlers...
         helpers.odoo_restart(2)
 
-    def _refresh_displays(self):
-        """Refresh all displays to hide the pairing code"""
-        for d in iot_devices:
-            if iot_devices[d].device_type == 'display':
-                iot_devices[d].action({
-                    'action': 'display_refresh'
-                })
```
Comment (MaxTheWhale) [inline addons/hw_drivers/connection_manager.py:69]: We don't need to do this anymore as the page is now fully reactive

**Review by loouis-t — APPROVED** [review]
LGTM, it's nice to finally get rid of jinja!

File: addons/hw_posbox_homepage/controllers/homepage.py [hunk 2]
```diff
@@ -41,6 +44,10 @@ def index(self):
     def logs_page(self):
         return http.Stream.from_path("hw_posbox_homepage/views/logs.html").get_response(content_security_policy=None)
 
+    @http.route('/status', auth='none', type='http')
+    def status_page(self):
+        return http.Stream.from_path("hw_posbox_homepage/views/status_display.html").get_response(content_security_policy=None)
```
[the remark this answers is not in the document]
Comment (MaxTheWhale) [reply addons/hw_posbox_homepage/controllers/homepage.py]: @xmo-odoo Good point, I've added a CSP for all the endpoints [here](https://github.com/odoo/odoo/pull/189909/files#diff-941fbc6d536b64fca65c8db9cb31a4b20df8318423d49b12446e8ea897306a70R33-R41)

**Review by xmo-odoo — COMMENTED** [review]
@robodoo override=ci/security

**Review by Ysoroko — APPROVED** [review]
@robodoo r+

**Comment by Ysoroko** [comment]
@robodoo rebase-ff

**Pull request closed without merging**